### PR TITLE
Create README for examples

### PR DESCRIPTION
**Pull request by timClicks**
Add instructions for readers who may not know how to run the example programs. I personally found this to be a little confusion when I was getting started.

### CODE REVIEW

File: sources/examples/README.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+OpenDylan example projects
+==========================
+
+Each sub-directory contains a standalone Dylan project. To compile them, you should have 
+a `working OpenDylan <https://opendylan.org/download/index.html>`_ installation.
```
Comment (cgay) [inline sources/examples/README.rst]: Open Dylan is generally written with a space.  (Here and in the title above.)

File: sources/examples/README.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Each sub-directory contains a standalone Dylan project. To compile them, you should have 
+a `working OpenDylan <https://opendylan.org/download/index.html>`_ installation. 
+
+To verify that your installation is working correctly, try checking the compiler's version::
+
+    $ dylan-compiler -version
+    Version 2019.1
+
+If a version string isn't appearing, ``dylan-compiler`` may not be on your ``$PATH``. For
```
Comment (cgay) [inline sources/examples/README.rst]: "isn't appearing" -> "doesn't appear"  :)

File: sources/examples/README.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+If a version string isn't appearing, ``dylan-compiler`` may not be on your ``$PATH``. For 
+help, check the `installation instructions <https://opendylan.org/download/index.html>`_ again 
+or reach out on one of the `community's communication channels <https://opendylan.org/community/index.html>`_.
+
+
+Compiling examples
+------------------
+
+To compile any of the examples, run ``dylan-compiler`` with the ``-build`` option. For any given project, 
+follow this is the pattern::
```
Comment (cgay) [inline sources/examples/README.rst]: following this pattern::

File: sources/examples/README.rst (hunk 1 above)
Comment (timClicks) [reply sources/examples/README.rst]: Fixed

File: sources/examples/README.rst (hunk 3 above)
Comment (timClicks) [reply sources/examples/README.rst]: Should be fixed. I've cut this sentence out.

**Comment by timClicks** [comment]
Recommend rebasing both commits together when merging :)

File: sources/examples/README.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+
+Where to start
+--------------
+
+Experiment with the examples in the ``console`` directory. Each of them generates one or more CLI utilities 
+that explore various features of the Dylan language.
+
+- ``console/hello-world`` demonstrates defining methods, printing to the console and not needing a "``main()`` function" for Dylan programs.
+- ``console/factorial`` demonstrates integer operations and composing libraries. ``factorial-big.lid`` and `factorial-small.lib`` use the same base code to produce multiple applications with different integer internals.
```
Comment (cgay) [inline sources/examples/README.rst]:
```
`factorial-small.lib``
```
should be
```
``factorial-small.lid``
```
Note missing backquote and 'b' should be 'd'.

Comment (cgay) [reply sources/examples/README.rst]: Tim, any plans to fix this?

Comment (cgay) [reply sources/examples/README.rst]: (There's whitespace at end of lines also.)

Comment (timClicks) [reply sources/examples/README.rst]: I feel terrible for leaving this hanging for so long. Fixed.

File: sources/examples/README.rst [hunk 5]
```diff
@@ -28,17 +28,17 @@ Executable files, if any, will appear in ``_build/bin/``.
...
 Where to start
 --------------
 
-Experiment with the examples in the ``console`` directory. Each of them generates one or more CLI utilities 
+Experiment with the examples in the ``console`` directory. Each of them generates one or more CLI utilities
 that explore various features of the Dylan language.
 
 - ``console/hello-world`` demonstrates defining methods, printing to the console and not needing a "``main()`` function" for Dylan programs.
-- ``console/factorial`` demonstrates integer operations and composing libraries. ``factorial-big.lid`` and `factorial-small.lib`` use the same base code to produce multiple applications with different integer internals.
+- ``console/factorial`` demonstrates integer operations and composing libraries. ``factorial-big.lid`` and `factorial-small.lid`` use the same base code to produce multiple applications with different integer internals.
```
Comment (cgay) [inline sources/examples/README.rst]: Sorry, but one last thing...  There should be two back ticks before factorial-small.lid, rather than one.

**Pull request merged**